os/dutils.sh

```diff
@@ -57,6 +60,58 @@ function FIND_DBGBINFILE()
 	KERNEL_DBG_BIN="tinyara$EXEEXT"
 }
 
+function TRAP_MENU()
+{
+	echo ======================================================
+	echo "  \"Run TRAP tool for\""
+	echo ======================================================
+	echo "  \"1. Existing binary\""
+	echo "  \"2. New binary\""
```

Comment:
I don't understand what they mean.
It looks that
1. existing -> use default path for elf
2. new -> give an elf path (not in default)

Reply:
Modified